src/main/java/data/JsonConverter.java

```
@@ -122,14 +122,42 @@ public String getConfigMap() {
     return gson.toJson(map);
   }
 
+  /**
+   * Converts a JSON array of Account objects to an ArrayList of Accounts
+   * @param accountArray JSON representation of array of account objects
+   * @return list of accounts
+   */
+  public static ArrayList<Account> buildAccountsFromJsonArray(JSONArray accountArray) {
+    ArrayList<Account> accounts = new ArrayList<Account>();
+    for (int i = 0; i < accountArray.length(); i++) {
+      accounts.add(new Account(accountArray.getJSONObject(i)));
+    }
+    return accounts;
+  }
+
   /** Return list of vendor IDs that exist in the filesystem.*/
   private ArrayList<String> getVendorIDs() {
     File root = new File(basePath);
-    return new ArrayList<>(Arrays.asList(Objects.requireNonNull(root.list())));
+    return new ArrayList<String>(Arrays.asList(Objects.requireNonNull(root.list())));
   }
 
-  /** TODO Return list of account IDs corresponding to a given vendor. */
-  private ArrayList<String> getAccountIDs(String vendorID) {
-    return new ArrayList<>(Arrays.asList("ECHO", "FOXTROT", "GOLF", "HOTEL"));
+  /**
+   * Returns an ArrayList of account ID strings.
+   * If none exist for the vendor, an empty ArrayList is returned
+   */
+  private ArrayList<String> getAccountIDs(String vendorID) throws IOException {
+    try {
+      String config = getConfig(vendorID);
+      Vendor vendor = new Vendor(config, vendorID);
+      ArrayList<Account> accounts = vendor.getAccounts();
+      ArrayList<String> accountIds = new ArrayList<String>();
+
+      for (Account account : accounts) {
+        accountIds.add(account.getAccountID());
+      }
+      return accountIds;
+    } catch (IOException e) {
+      return null;
```

Maybe throw an exception here? Fine to add a TODO for now